corehq/apps/tzmigration/planning.py

```diff
@@ -92,7 +92,7 @@ def connect():
         self.domain = domain
         self.readonly = readonly
         self.db_filepath = db_filepath
-        self.engine = create_engine("sqlite://", creator=connect)
+        self.engine = create_engine(f"sqlite:///{db_filepath}", creator=connect)
```

Comment:
I guess making `self.engine` a file path makes `self.engine.pool` not a `SingletonThreadPool`? Curious how/why, but I'm guessing that `SingletonThreadPool` is the default for in-memory dbs, but not file dbs?

Reply:
> `SingletonThreadPool` is the default for in-memory dbs, but not file dbs?

This is correct. See https://docs.sqlalchemy.org/en/13/dialects/sqlite.html#threading-pooling-behavior